src/css/base/typography.scss

```diff
@@ -3,9 +3,14 @@
 
 body {
   color: $color-base;
+  font-feature-settings: 'kern';
```

Comment:
What are all these new settings doing? Some of them do not seem cross-browser. What is there browser support?